### PR TITLE
Rev 65 homepage upcoming tab slides are broken

### PR DESCRIPTION
**Pull request by IbrahimAljabrST**
### Jira URL

https://optimumpartners.atlassian.net/browse/REV-65

### Description

home page upcoming slider shows two slides 

### Dependencies in Other PRs,Repo

none

### Testing Instructions

-check the home page events slider

### CODE REVIEW

File: src/components/home/Events/data.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const data = [
     img: chairs,
     place: 'somewhere',
     time: '7:30 - 10:30PM',
-    title: 'Lorem Ipsum',
+    title: 'Lorem Ipsum111',
```
Comment (mohdabed123) [inline src/components/home/Events/data.js:15]:
```suggestion
    title: 'Lorem Ipsum',
```

File: src/components/home/Events/data.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ const data = [
     img: peoplegroup,
     place: 'somewhere',
     time: '7:30 - 10:30PM',
-    title: 'Lorem Ipsum',
+    title: 'Lorem Ipsum2222',
```
Comment (mohdabed123) [inline src/components/home/Events/data.js:26]:
```suggestion
    title: 'Lorem Ipsum 2',
```

File: src/components/home/Events/data.js [hunk 3]
```diff
@@ -34,18 +34,18 @@ const data = [
     img: christmas,
     place: 'somewhere',
     time: '7:30 - 10:30PM',
-    title: 'Lorem Ipsum',
+    title: 'Lorem Ipsum333',
```
Comment (mohdabed123) [inline src/components/home/Events/data.js:37]:
```suggestion
    title: 'Lorem Ipsum 3',
```

**Review by mohdabed123 — APPROVED** [review]
LGTM but one minor change can we

**Review by AhmadAlShwaiki — APPROVED** [review]

**Pull request merged**